docs/source/user/quickstart.rst

```diff
@@ -27,7 +27,7 @@ in file ``examples/example.py``
 .. code-block:: python
 
     core.Context.create(
-        username='{usr}', password='{pwd}', host='{decantercoreserver}')
+        username='gp', password='gp-admin', host='http://localhost:3000')
```

Comment:
use this on document: "username='{usr}', password='{pwd}', host='{host:port}')"

Reply:
done